app/controllers/answers_controller.rb

```diff
@@ -1,5 +1,5 @@
 class AnswersController < ApplicationController
-  before_filter :require_user
+  before_action :require_user
```

Comment:
Thank you for taking care of this. The warning from RSpec was almost intolerable. Glad this will work still if it gets upgraded to Rails 5.1